.github/checkgroup.yml

```diff
@@ -143,13 +143,13 @@ subprojects:
       - "build-cuda (3.9, 1.13, 11.8.0)"
       - "build-cuda (3.9, 1.13, 12.0.1)"
       - "build-cuda (3.10, 2.0, 11.8.0)"
-      - "build-cuda (3.10, 2.0, 12.0.1)"
```

Comment:
If you remove this config from checkgroup, it should also be removed from `.github/workflows/docker-build.yml`

Reply:
yes minor oversight
done